.github/workflows/build.yml

```diff
@@ -0,0 +1,282 @@
+
+name: 111 Adaptor Build Workflow
+on:
+    pull_request:
+        types: [opened, synchronize, reopened]
+        branches:
+            - master
+    push:
+        branches:
+            - master
```

Comment:
Should probably rename the master branch to main to be consistent with the other repos at some point.

Reply:
ok, I can do that in a separate PR